python/cac_tripplanner/cms/models.py

```diff
@@ -1,9 +1,14 @@
+from ckeditor.fields import RichTextField
```

Comment:
I thought this was handled as:
built-in modules
non-django third-party modules
django modules
django third-party modules
local source

Either way, we should move the discussion off line and clarify whatever we choose in the coding guidelines.